package.json

```diff
@@ -59,7 +59,8 @@
     "reboot": "yarn clean && yarn reset && yarn install && yarn start",
     "rollup.bundle": "rollup -c -m",
     "rollup.watch": "yarn rollup -w",
-    "rollup": "yarn rollup.bundle"
+    "rollup": "yarn rollup.bundle",
```

Comment:
possible dumb question:

Are we using webpack for Storybook and rollup for the production package?

Reply:
Maybe this should go in the Q&A on this repo...

Reply:
@himerus would have an answer for you on this, I'm not 100% sure.

Reply:
Yes, Webpack currently for Storybook until the faster bundlers like vite/snowpack are more stable. 
Storybook directly can consume the .ts stuff so no compiling is needed to see updates during development. 
Rollup for the production (bundled) build and standard tsc for the build that has the components broken out.